gradle/libs.versions.toml

```diff
@@ -9,6 +9,7 @@ androidx-activity = '1.9.3'
 androidx-annotation = '1.9.1'
 androidx-appcompat = '1.7.0'
 androidx-arch-core = '2.2.0'
+androidx-browser = '1.5.0'
```

Comment:
Note that this is an outdated version of the library. The latest is [1.8.0](https://developer.android.com/jetpack/androidx/releases/browser#1.8.0).